src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private Entry<K, V>[] items;
+    private int length = 0;
+
+    public StorageImpl() {
+        this.items = new Entry[MAX_LENGTH];
+    }
+
     @Override
     public void put(K key, V value) {
+        Entry<K, V> entry = new Entry<>(key, value);
+        for (int i = 0; i < length; i++) {
+            if ((items[i].getKey() == null && key == null)
```

Comment:
```suggestion
            if ((items[i].getKey() == key)
```
тут можна спростити перевірку (якщо два значення нал їх можна по посиланню порівнювати)